.github/workflows/ci.yml

```diff
@@ -14,6 +28,16 @@ jobs:
     runs-on: self-hosted
     steps:
     - uses: actions/checkout@v2
+    - name: License Check
+      run: |
+        set -x
+        git init
+        git add --all
+        git -c user.name="CI Bot" -c user.email="<>" commit -m "initial state"
```

Comment:
does `addlicense` need to make a commit? is that why you need this?

Reply:
It doesn't need to commit, just see the status. 

I wrote this check a while ago for Bank of Anthos. I assume I got an error at some point when this was left out, but I can double check to see if it still works with this line removed

Reply:
Actually, I remember the context now: there is no git repo in the filesystem here originally, so I needed to create one and make an initial commit as a way of checking for changes after running the `addlicence` command.

It's slightly hacky, but this was the cleanest solution I could think of. If you have other ideas for how to compare file system states, I'm open to other ideas